packages/template-retail-react-app/app/translations/en-US.json

```diff
@@ -174,6 +174,9 @@
   "cart.info.removed_from_cart": {
     "defaultMessage": "Item removed from cart"
   },
```

Comment:
TODO: We'll need to send this en-US file to our translators, so they will be able to translate it to other languages.

You don't need to tell them specifically what has been added, replaced, etc. Because from what I heard, their tools are able to automatically detect the changes.